apis/api-journeys/src/workers/migrations/subscribe-blocks/replace-subscribe-blocks.ts

```
@@ -0,0 +1,251 @@
+// Migration script to replace Subscribe blocks
+import { v4 as uuidv4 } from 'uuid'
+
+import { PrismaClient } from '.prisma/api-journeys-client'
```

ESLint

[error] Useless path segments for ".prisma/api-journeys-client", should be "./.prisma/api-journeys-client"

(import/no-useless-path-segments)